_posts/samus_returns/2024-10-25-decrypted-copy.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "How to dump a decrptyed copy of Metroid: Samus Returns"
+tags: samus_returns
+---
+Using Godmode9 from your 3DS, you can dump a decrypted copy of the game. The end result will provide you with one of the following files: .3ds, .cia, .app, .cxi.
```

Comment:
having an answer for the following would be nice as an end user:
- i have a game cartridge, should i use 3ds or cia? does it matter which one i use?
- i have a digital copy, should i use cia, app or cxi? does it matter which one i use?